backend/src/tools/api.py

```
@@ -1,86 +1,51 @@
+
+This module provides a simplified interface for API tool creation and OpenAPI
+specification handling by importing and re-exposing functionality from
+specialized submodules.
+
+This maintains backward compatibility while providing better code organization.
+"""
+
+# Import all functions from specialized modules to maintain backward compatibility
+from .openapi_spec import (
+    get_api_spec,
+    openapi_from_url,
```

Ruff (0.12.2)

`.openapi_spec.openapi_from_url` imported but unused

Remove unused import

(F401)